<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>852</width>
    <height>573</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true">StarBackup</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/icons/star</normaloff>:/icons/star</iconset>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LeftToRight</enum>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="ScrollAreaWithWheels" name="scrollArea_2">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>1</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>150</width>
         <height>80</height>
        </size>
       </property>
       <property name="widgetResizable">
        <bool>true</bool>
       </property>
       <widget class="BackupListWidget" name="backupListWidget">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>827</width>
          <height>466</height>
         </rect>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </widget>
      <widget class="QStackedWidget" name="logs">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>2</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>852</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionImport_Backup"/>
    <addaction name="actionExport_Backup"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menu_Options">
    <property name="title">
     <string>&amp;Backup</string>
    </property>
    <addaction name="actionAdd_Profile"/>
    <addaction name="actionEdit_backup"/>
    <addaction name="actionRemove_backup"/>
    <addaction name="separator"/>
    <addaction name="actionStart_backup"/>
    <addaction name="actionStart_all_backups"/>
   </widget>
   <widget class="QMenu" name="menuOptions">
    <property name="title">
     <string>Options</string>
    </property>
    <addaction name="actionChange_language"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menu_Options"/>
   <addaction name="menuOptions"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="iconSize">
    <size>
     <width>36</width>
     <height>36</height>
    </size>
   </property>
   <property name="floatable">
    <bool>true</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionAdd_Profile"/>
   <addaction name="actionEdit_backup"/>
   <addaction name="actionRemove_backup"/>
   <addaction name="separator"/>
   <addaction name="actionStart_backup"/>
   <addaction name="actionStart_all_backups"/>
   <addaction name="separator"/>
   <addaction name="actionShow_logs"/>
   <addaction name="actionHide_logs"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionAdd_Profile">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/add</normaloff>:/icons/add</iconset>
   </property>
   <property name="text">
    <string>Add Backup</string>
   </property>
   <property name="toolTip">
    <string>Add a new backup profile</string>
   </property>
  </action>
  <action name="actionRemove_backup">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/remove</normaloff>:/icons/remove</iconset>
   </property>
   <property name="text">
    <string>Remove Backup</string>
   </property>
   <property name="toolTip">
    <string>Remove the selected backup</string>
   </property>
  </action>
  <action name="actionEdit_backup">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/edit</normaloff>:/icons/edit</iconset>
   </property>
   <property name="text">
    <string>Edit Backup</string>
   </property>
   <property name="toolTip">
    <string>Edit the selected backup</string>
   </property>
  </action>
  <action name="actionStart_backup">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/launch</normaloff>:/icons/launch</iconset>
   </property>
   <property name="text">
    <string>Start Backup</string>
   </property>
   <property name="toolTip">
    <string>Start the selected backup now</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/quit</normaloff>:/icons/quit</iconset>
   </property>
   <property name="text">
    <string>Quit</string>
   </property>
   <property name="toolTip">
    <string>Quit completely (scheduled backups will NOT be executed)</string>
   </property>
  </action>
  <action name="actionStart_all_backups">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/launchall</normaloff>:/icons/launchall</iconset>
   </property>
   <property name="text">
    <string>Start all backups</string>
   </property>
   <property name="toolTip">
    <string>Start all backups now</string>
   </property>
  </action>
  <action name="actionShow">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/window</normaloff>:/icons/window</iconset>
   </property>
   <property name="text">
    <string>Show</string>
   </property>
   <property name="toolTip">
    <string>Show the main window</string>
   </property>
  </action>
  <action name="actionShow_logs">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/2right</normaloff>:/icons/2right</iconset>
   </property>
   <property name="text">
    <string>Show logs</string>
   </property>
   <property name="toolTip">
    <string>Show the logs</string>
   </property>
   <property name="visible">
    <bool>true</bool>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionHide_logs">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/2left</normaloff>:/icons/2left</iconset>
   </property>
   <property name="text">
    <string>Hide logs</string>
   </property>
   <property name="toolTip">
    <string>Hide the logs</string>
   </property>
  </action>
  <action name="actionImport_Backup">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/download</normaloff>:/icons/download</iconset>
   </property>
   <property name="text">
    <string>Import Backup(s)</string>
   </property>
   <property name="toolTip">
    <string>Import some Backup Profile(s) from some XML file(s)</string>
   </property>
  </action>
  <action name="actionExport_Backup">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/upload</normaloff>:/icons/upload</iconset>
   </property>
   <property name="text">
    <string>Export Backup</string>
   </property>
   <property name="toolTip">
    <string>Export the selected Backup Profile to an XML File</string>
   </property>
  </action>
  <action name="actionChange_language">
   <property name="text">
    <string>Change language</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>BackupListWidget</class>
   <extends>QWidget</extends>
   <header>backuplistwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ScrollAreaWithWheels</class>
   <extends>QScrollArea</extends>
   <header>backuplistwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
